librz/core/cmd_descs/cmd_analysis.yaml

```diff
@@ -383,6 +383,104 @@ commands:
                     type: RZ_CMD_ARG_TYPE_RZNUM
                   - name: addr
                     type: RZ_CMD_ARG_TYPE_RZNUM
+      - name: afl
+        summary: List functions
+        subcommands:
+          - name: afl
+            summary: List all functions
+            cname: analysis_fcn_list
+            type: RZ_CMD_DESC_TYPE_ARGV_STATE
+            modes:
+              - RZ_OUTPUT_MODE_STANDARD
```

Comment:
I'm becoming more and more doubtful about the "STANDARD" mode for most table-like outputs, because I think the table is usually clearer about the meaning of each column (and more powerful). What do you think about it?

Reply:
You have a good point, but for now I think we should keep it as is, until we convert everything to the rzshell, and only then switch everything that is relevant to use table output as the default one.